articles/clients/grant-types/client-grant-types.md

```diff
@@ -0,0 +1,124 @@
+---
+  description: Using the Grant Types property on Clients
+  toc: true
+---
+
+# Client Grant Types
+
+Depending on your use case, Auth0 provides many authorization flows that suit your needs. As such, you'll need to track the type of flow you're using for a given client. As such, Auth0 includes a `grant_type` property on each Client for this purpose.
+
+## Grant Types
+
+The following is a list of grant types valid for Auth0 Clients. Each of the items represents one of the three types of authorization flows.
+
+There are three possible types of authorization flows:
+
+* Auth0-customized;
+* Legacy;
+* OIDC specification-conformant.
+
+The following is a list of Auth0-customized/OIDC specification-conformant `grant_types` that currently exist for use in Auth0:
+
+* `implicit`
+* `authorization_code`
+* `client_credentials`
+* `password`
+* `refresh_token`
+* `http://auth0.com/oauth/grant-type/password-realm`
+* `http://auth0.com/oauth/grant-type/mfa-oob`
+* `http://auth0.com/oauth/grant-type/mfa-otp`
+* `http://auth0.com/oauth/grant-type/mfa-recovery-code`
+
+The following is a list of legacy `grant_types`:
+
+* `http://auth0.com/oauth/legacy/grant-type/ro`
+* `http://auth0.com/oauth/legacy/grant-type/ro/jwt-bearer`
+* `http://auth0.com/oauth/legacy/grant-type/delegation/refresh_token`
+* `http://auth0.com/oauth/legacy/grant-type/delegation/id_token`
+* `http://auth0.com/oauth/legacy/grant-type/access_token`
+
+## Edit the `grant_type` Property
+
+To edit the `grant_type` property for your Auth0 Client, you'll need to make the appropriate call to the [Update a Client endpoint](/api/management/v2#!/Clients/patch_clients_by_id) of the [Management API](/api/management/v2).
+
+:::panel-warning Legacy Grant Types
+New Auth0 customers **cannot** add *any* of the legacy grant types to their Clients. Only customers as of TBD can add legacy grant types to their existing Clients.
+:::
+
+Auth0 requires the `grant_type` property, so using a Client *without* this property populated results in errors.
+
+### Existing Clients
+
+For existing Clients, Auth0 will populate the `grant_type` property with the correct value so that this change doesn't affect functionality.
```

Comment:
Something like this might make it clearer that we are added ALL grant types to clients...

In order to avoid breaking existing customers, all existing Auth0 clients for all customers as of TBD will be populated with all Auth0 legacy, Auth0 extension, and spec-formant grant_types.